.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,4 @@
-### Acceptance Criteria
-- Include here all things that this PR should solve
+Please go the the `Preview` tab and select the appropriate Pull Request template:
 
-
-### Security Checklist
-- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.
+* [Feature Branch](?expand=1&template=feature_branch_pr_template.md) - Use this PR template when you are merging a feature branch into `dev`
+* [Version Bump](?expand=1&template=version_bump_pr_template.md) - Use this PR template when you are merging a version bump branch into `dev`
```

Comment:
It's not exactly when you are merging a branch, it's when you are creating a pull request of a feature branch / version bump.

Reply:
I agree, done in aeb55c50473c481acf71bfa25284f3233ccbfde2

Reply:
Updated the text to be less repetitive in 6c728840a6320d744fcba7207ad68dae69edd281